BowerToNPM.java

```diff
@@ -0,0 +1,159 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.StringBuilder;
+import java.nio.file.Path;
```

Comment:
Thanks! I've updated the `runme` with comments. Yeah, having a single Java file seems arbitrary.

While it would be nice for this to work across all systems, I've decided to put this on the back burner and move forward with testing the server. I'll update the `README` to note that `BowerToNPM` won't work on Windows.